### PR TITLE
Load sample data from empty state

### PR DESCRIPTION
**Pull request by CtrlSpice**
Add functionality to generate sample data on the server, load it into the `TraceStore` and give the okay to refresh the page.

Here it is working!

LoadSampleData

### CODE REVIEW

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 1]
```diff
@@ -17,9 +16,47 @@ import {
   OrderedList,
   Stack,
   Text,
+  useBoolean,
   useColorModeValue,
 } from "@chakra-ui/react";
 
+async function loadSampleData() {
+  let response = await fetch("/api/sampleData");
+  console.log(response.status);
```
Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:25]: nit: remove logging before merge

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 2]
```diff
@@ -17,9 +16,47 @@ import {
...
+      colorScheme="pink"
+      onClick={() => {
+        setIsLoading.on;
+        loadSampleData();
+      }}
+    >
+      Load Sample Data
+    </Button>
+  );
+}
```
Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:58]: Nice and clean!

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 3]
```diff
@@ -17,9 +16,47 @@ import {
...
+        spinnerPlacement="start"
+      />
+    );
+  }
+
+  return (
+    <Button
+      colorScheme="pink"
+      onClick={() => {
+        setIsLoading.on;
```
Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:51]: Should this not be `setIsLoading.on()`?

File: desktop-exporter/sample_data.go [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+	traceData := ptrace.NewTraces()
+
+	traceData.ResourceSpans().EnsureCapacity(1)
+
+	// Resource data for currencyservice
+	currencyResourceSpan := traceData.ResourceSpans().AppendEmpty()
+	currencyResource := currencyResourceSpan.Resource()
+	currencyResource.SetDroppedAttributesCount(0)
+	currencyResource.Attributes().PutStr("service.name", "SAMPLE: currencyservice")
+	currencyResource.Attributes().PutStr("telemetry.sdk.language", "cpp")
```
Comment (jmorrell) [inline desktop-exporter/sample_data.go:23]: all the cool kids use Rust these days I hear

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-02-01 19 35 47 - a corgi looking at a charcuterie board, digital art

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx (hunk 3 above)
Comment (CtrlSpice) [reply desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:51]: Yep. Sorry.

File: desktop-exporter/sample_data.go (hunk 4 above)
Comment (CtrlSpice) [reply desktop-exporter/sample_data.go:23]: Currency conversion isn't cool tho.

**Pull request merged**